test/legacy/data/test_batch.py

```diff
@@ -1,7 +1,7 @@
 import torch
-import torchtext.data as data
+import torchtext.legacy.data as data
 
-from ..common.torchtext_test_case import TorchtextTestCase
+from ...common.torchtext_test_case import TorchtextTestCase
```

Comment:
Triple "." is probably not intended here

Reply:
It needs to jump two folders to access `TorchtextTestCase` class